aliparadise/Dose
# Completed detail view for patient medications

closes #4
Changes Made

1. Replaced file `ManageMeds.cshtml` with `MedicationDetails.cshtml`.
2. Changed `PatientController.cs` ManageMeds method to MedicationDetails method for D.R.Y. code.

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout ap-patientMedicationDetails
   ```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Test app functionality.
   >If an authorized user is signed in, they should navigate to their patient list but clicking the patient link in the navbar. They should see a list of their patients and they should click on the Manage Meds link for a patient. They should then be taken to the Medication Details view to see the patient medication details of each medication.
